cypress/plugins/index.js

```diff
@@ -0,0 +1,12 @@
+const renameKeys = o =>
+  Object.keys(o || {}).reduce(
+    (acc, key) =>
+      Object.assign({}, acc, {[key.replace(/^CYPRESS_/, '')]: o[key]})
```

Comment:
Doesn't this work with the spread operator too?

Reply:
Nope. not supported in this context